server/datastore/mysql/software.go

```diff
@@ -3529,7 +3529,12 @@ func (ds *Datastore) ListHostSoftware(ctx context.Context, host *fleet.Host, opt
 				vpp_apps.latest_version AS vpp_app_version,
 				vpp_apps.platform as vpp_app_platform,
 				NULLIF(vpp_apps.icon_url, '') as vpp_app_icon_url,
-				vpp_apps_teams.self_service AS vpp_app_self_service
+				vpp_apps_teams.self_service AS vpp_app_self_service,
+				host_software.software_id AS software_id,
+				host_software.last_opened_at,
+				software.source AS software_source,
+				software.version AS version,
+				software.bundle_identifier AS bundle_identifier
```

Comment:
I don't think we technically need these, because I removed the `hostVPPInstalledTitles[s.ID] = s`. I do wonder if I should delete this just in case so it's not confusing.

Reply:
Yes please.